### ISSUE TITLE
Hold BackSpace key crash application

### ISSUE THREAD

**Issue by vit1251** [askovpen/gossiped#117]
**To Reproduce**

1. Start application
2. Select EchoMail area
3. Reply 
4. Hold BackSpace key

```
panic: runtime error: slice bounds out of range [:-1] [recovered]
	panic: runtime error: slice bounds out of range [:-1]

goroutine 1 [running]:
github.com/rivo/tview.(*Application).Run.func1(0xc0000a8580)
	/root/go/pkg/mod/github.com/rivo/tview@v0.0.0-20190626133940-2e907d29e475/application.go:149 +0x82
panic(0x62bca0, 0xc0004ba280)
	/usr/local/go/src/runtime/panic.go:679 +0x1b2
github.com/askovpen/gossiped/pkg/ui.(*EditBody).InputHandler.func1.3()
	/opt/gossiped/pkg/ui/editbody.go:541 +0x5e4
github.com/askovpen/gossiped/pkg/ui.(*EditBody).InputHandler.func1(0xc00000c560, 0xc00005cd60)
	/opt/gossiped/pkg/ui/editbody.go:618 +0x3dd
github.com/rivo/tview.(*Box).WrapInputHandler.func1(0xc00000c560, 0xc00005cd60)
	/root/go/pkg/mod/github.com/rivo/tview@v0.0.0-20190626133940-2e907d29e475/box.go:152 +0x5d
github.com/rivo/tview.(*Application).Run(0xc0000a8580, 0x0, 0x0)
	/root/go/pkg/mod/github.com/rivo/tview@v0.0.0-20190626133940-2e907d29e475/application.go:234 +0x415
github.com/askovpen/gossiped/pkg/ui.(*App).Run(0xc00008b560, 0x1, 0x1)
	/opt/gossiped/pkg/ui/app.go:34 +0x56
main.main()
	/opt/gossiped/gossiped.go:38 +0x2a7
```

**Comment by askovpen** [comment]
plz check

**Comment by askovpen** [comment]
fix in #118
